compiler/dyno/include/chpl/framework/Context.h

```diff
@@ -247,6 +255,22 @@ class Context {
   Context();
   ~Context();
 
+  /**
+    Store a string corresponding to CHPL_HOME in the context. The context does
+    not take ownership of the string, so the argument should live at least as
+    long as the context itself.
+
+    Changing chplHome invalidates previous caches of printchplenv output.
+  */
+  void setChplHome(const char* chplHome);
```

Comment:
Should this instead be something we can do when constructing the Context? That would allow you to stop worrying about it changing and invalidating the `getChplEnv` result.